src/frontend/komponenter/Oppgaver/visoppgave.less

```diff
@@ -0,0 +1,99 @@
+.visoppgaver {
+    padding: 1rem;
+}
+
+.filterskjema {
+    display: flex;
+    flex-direction: column;
+
+    &__content {
+        margin-top: 2rem;
+    }
+
+    &__filtre {
+        display: flex;
+        &__input{
+            padding-right: 2rem;
+        }
+    }
+
+    &__button {
+        width: 10rem;
+    }
+    
+    &__tilbakestill {
+        margin-left: 2rem;
+    }
+}
+
+table.tabell{
+    width: 1900px;
+    margin-top: 50px;
+    table-layout: fixed;
+    border-collapse: collapse;
+}
+
+table.tabell tbody{
+    display:block;
+    width: 100%;
+    overflow: auto;
+    height: 30rem;
+}
+
+table.tabell thead tr{
+    display:block;
+}
+
+table.tabell th, table.tabell td{
+    text-align: left;
+}
+
+.regdato {
+    width: 135px;
+}
+
+.oppgavetype {
+    width: 215px;
+}
+
+.gjelder {
+    width: 150px;
+}
+
+.frist {
+    width: 90px;
+}
+
+.prioritet {
+    width: 120px;
+}
+
+.beskrivelse {
+    width: 318px;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    padding: 0;
+}
+
+.bruker {
+    width: 135px;
+}
+
+.enhet {
+    width: 155px;
```

Comment:
Maybe use `min-width` for the columns so that the table is not "fixed" size?

Reply:
agree